src/sentry/static/sentry/app/views/organizationActivity/activityFeedItem.jsx

```diff
@@ -73,7 +73,7 @@ class ActivityItem extends React.Component {
 
     const versionLink = data.version ? (
       <VersionHoverCard orgId={orgId} projectId={project.slug} version={data.version}>
-        <Version version={data.version} orgId={orgId} />
+        <Version version={data.version} orgId={orgId} projectId={project.id} />
```

Comment:
@matejminar Should `preserveGlobalSelection` be here as well?

Reply:
Hey @dashed, please see the updated PR, because I decided to revamp it a bit more.